appengine/flexible/multiple_services/api_gateway/README.md

```diff
@@ -0,0 +1,56 @@
+# Python Google Cloud Microservices Example - API Gateway
+
+This example demonstrates how to deploy multiple python services to [App Engine flexible environment](https://cloud.google.com/appengine/docs/flexible/)
+
+## To Run Locally
```

Comment:
OK.  Needs the other changes though, and testing with deployed code.  Will push up now so you can see and again when changes complete.